source/conf.py

```diff
@@ -31,7 +30,33 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.autodoc'
+    'autodoc2',
+    'myst_parser',
+    'sphinx_rtd_theme',
+]
+
+myst_enable_extensions = ['colon_fence', 'fieldlist']
+
+autodoc2_packages = [
+    {
+        'path': '../ibmcloudant',
+        'exclude_files': [
+            'common.py',
+            'version.py',
+            'cloudant_base_service.py',
+            'couchdb_session_get_authenticator_patch.py'
+        ]
```

Comment:
The session token manager is internal, so I think we can probably exclude it here as well.
```suggestion
            'couchdb_session_get_authenticator_patch.py',
            'couchdb_session_token_manager.py'
        ]
```